src/ethereum/api/fragments.ts

```diff
@@ -53,9 +53,6 @@ export const thirdPartyFragment = () => gql`
   fragment thirdPartyFragment on ThirdParty {
     id
     managers
-    isApproved
-    maxItems
-    totalItems
```

Comment:
Unless we're creating another endpoint to get the max items and the total items, I would leave these properties here so we can use them in the UI to tell the user to buy more slots.